tests/e2e-test-framework/framework/ssh.py

```diff
@@ -46,6 +46,6 @@ def exec(self, command: str) -> Tuple[str, List[Any]]:
         ssh_client.close()
 
         if len(error) > 0:
-            logging.error(f"SSH command {command} failed: {error}")
+            logging.warning(f"SSH command {command} failed: {error}")
```

Comment:
FYI: reduced the logging level, because interrupting `dd` returns a status code different than 0